torch/csrc/utils/schema_info.h

```diff
@@ -15,10 +16,38 @@ namespace utils {
 
 struct TORCH_API SchemaInfo : c10::FunctionSchema {
  public:
-  explicit SchemaInfo(c10::FunctionSchema schema) : FunctionSchema(schema) {}
+  explicit SchemaInfo(c10::FunctionSchema schema)
+      : FunctionSchema(schema), updated_(false) {}
   explicit SchemaInfo(const char* signature)
-      : FunctionSchema(torch::jit::getOperatorForLiteral(signature)->schema()) {
-  }
+      : FunctionSchema(torch::jit::getOperatorForLiteral(signature)->schema()),
+        updated_(false) {}
+
+  bool is_mutable();
+
+  bool is_mutable(size_t index);
+
+  bool is_mutable(c10::string_view name);
+
```

Comment:
Note that these will need to be marked as `override` and the base class functions marked as `virtual` for this subclass to be used as intended.